src/components/sidebar/right-sidebar/new-datepicker/NewDateSlider.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import { Slider, Rail, Handles, Tracks } from "react-compound-slider";
+import { Handle, Track, TooltipRail } from "./components"; // example render components - source below
+import {useEffect, useState} from "react";
+
+const sliderStyle: React.CSSProperties = {
+  position: "relative",
+  width: "80%",
+  margin: "5% 10%"
+};
+
+interface DateSliderProps {
+  minPossibleValue: number; //minimum possible value for slider
+  maxPossibleValue: number; //maximum possible value for slider
+  values: number[];
+  minDate: Date;
+  onMouseUp: Function;
```

Comment:
Can we either add a comment describing onMouseUp's function signature or add some typing that will describe it better?